files/zh-cn/web/api/highlight/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Highlight
+slug: Web/API/Highlight
+l10n:
+  sourceCommit: 47ed48a36b456f8ea9ab6aaa5969c55d2912edcb
+---
+
+{{APIRef("CSS Custom Highlight API")}}
+
+[CSS Custom Highlight API](/zh-CN/docs/Web/API/CSS_Custom_Highlight_API) 的 **`Highlight`** 接口用于表示使用此 API 在文档上设置样式的 {{domxref("Range")}} 集合。
+
+要为页面中的任意范围添加样式，可实例化一个新的 `Highlight` 对象，向其中添加一个或多个 `Range` 对象，然后使用 {{domxref("HighlightRegistry")}} 注册该对象。
+
+`Highlight` 是一个 [类 Set 对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Set#set-like_browser_apis)，可以容纳一个或多个 `Range` 对象。
+
+{{InheritanceDiagram}}
+
+## 构造器
+
+- {{domxref("Highlight.Highlight()", "Highlight()")}}
+  - : 返回新创建的 `Highlight` 对象。
+
+## 实例属性
+
+_`Highlight` 接口不继承任何属性。_
+
+- {{domxref("Highlight.priority")}}
+  - : 表示此 `Highlight` 对象优先级的数字。当多个高亮重叠时，浏览器会根据该优先级来决定如何为重叠部分添加样式。
+- {{domxref("Highlight.size")}} {{ReadOnlyInline}}
+  - : 返回 `Highlight` 对象中范围的数量。
```

Comment:
I apologize for any confusion. The term "range" does not have a specific translation in the reference this document mentioned. If there is any additional information or context that you can provide, please remind me.
```suggestion
  - : 返回 `Highlight` 对象中 range 的数量。
```